### PR TITLE
Clarify installation for 64-bit and 32-bit prefixes.

### PR DESCRIPTION
**Pull request by shmerl**
More details for manual installation.

### CODE REVIEW

**Comment by wu-yue** [comment]
Maybe it would be better to explicitly point out that the syswow64 and system32 would swap under different arch. Took me sometime to realize it.

**Comment by wael444** [comment]
plz merge because the one in README doesnt work but this clarifies what to actually do

File: README.md [hunk 1]
```diff
@@ -9,14 +9,25 @@ The most recent development builds can be found [here](https://github.com/doitsu
 Release builds can be found [here](https://github.com/doitsujin/dxvk/releases).
 
 ## How to use
-In order to install a DXVK package obtained from the [release](https://github.com/doitsujin/dxvk/releases) page into a given wine prefix, copy or symlink the DLLs into the following directories as follows, then open `winecfg` and manually add DLL overrides for `d3d11`, `d3d10core`, `dxgi`, and `d3d9`:
+In order to install a DXVK package obtained from the [release](https://github.com/doitsujin/dxvk/releases) page into a given wine prefix, copy or symlink the DLLs into the Wine prefix system directories as shown below, then open `winecfg` and manually add DLL overrides for `d3d11`, `d3d10core`, `dxgi`, and `d3d9`.
```
Comment (wael444) [inline README.md:12]: it's not exactly clarified if native or builtin overrides are supposed to be added, setup_dxvk does native.

Comment (shmerl) [reply README.md:12]: OK, may be I can simply add an actual command that does the override to simplify this and clarify it in the text as well.

**Pull request closed without merging**